Correct html PropTypes

f-garofalo opened the pull request:
Correct html PropTypes

It's string, not bool

as79-alex committed:
Adoption for sweetalert2

as79-alex committed:
Readme update and remove comments

as79-alex committed:
Readme update

as79-alex committed:
Readme update and css files

as79-alex committed:
Readme update and css files

alex-shamshurin committed:
Update README.md

hipstersmoothie committed:
allow swal instance to be passed in

alex-shamshurin committed:
Merge pull request #5 from hipstersmoothie/prop-swal

Allow swal instance to be passed in

hipstersmoothie committed:
Function that accepts swal instance and returns the component (#6)

* default funciton accepts swal instance and returns the component

* don't mess with default export

* should default to swal provided in jsx if one isn't specified in withSwal

* bind the argument in a way that will work

* change name to asReactComponent

* unused variable

* still allow swal to be overriden

* move to function that returns a class

as79-alex committed:
Version bump

as79-alex committed:
React update to 16

as79-alex committed:
SweetAlert and other deps upgrade
Yarn instead of npm in scripts

as79-alex committed:
Readme.md update

zenflow commented:
@f-garofalo Look at what you're doing... This repo is sweetalert-react, not sweetalert2-react...

zenflow commented:
@f-garofalo And I don't see any commits either (1) by you, or (2) correcting prop types

franciscop-invast committed:
Fixed html being a string (#12)

Commit:
Changed html default value to null (#22)

alex-shamshurin committed:
Babel upgraded

alex-shamshurin committed:
Deps update, support for latest sweetAlert

alex-shamshurin committed:
Merge branch 'feature/upgrade' into develop

alex-shamshurin committed:
Merge branch 'release/0.8.0'

alex-shamshurin committed:
Added dist folder to package.json

alex-shamshurin committed:
Merge branch 'release/0.8.1'

alex-shamshurin committed:
Merge branch 'release/0.8.1' into develop

dominicrico committed:
Pass result and reason to callbacks (#26)

alex-shamshurin committed:
Version bump to 0.8.2

alex-shamshurin committed:
Merge branch 'release/0.8.2'

jcoon97 committed:
[#28]: Fixes typo in onConfirm(m) that was preventing event handlers from being called (#29)

dominicrico committed:
fixes #23 (#25)

* fixes #23

Promise will not be rejected if modal is closed, therefor we must check the result value

* improves tests

* removes unused code and console.logs

* merged

* reverts merge

jcoon97 committed:
[#20]: Adds a few new customizable attributes to SweetAlerts, such as backdrop, position, and toast. You can learn more about each of the aforementioned properties on the official sweetalerts website. (#30)

JackHowa committed:
Run react codemod on lifecycle methods (#33)
